web/src/components/feature/calculation_input/CalculationInput.tsx

```diff
@@ -91,6 +91,16 @@ export const CalculationInput = ({
     if (mode === 'edit' && selectedPackage !== undefined) {
       oldPackages = oldPackages.filter((x) => x.name !== selectedPackage.name)
     }
+
+    const packageNameConflict = oldPackages.find(
+      (x) => x.name === newPackage?.name
+    )
+    if (mode === 'new' && packageNameConflict) {
```

Comment:
What if you're renaming an existing package to a name that already exists? Don't you want a warning then before overwriting the other package?

Reply:
Good point!